root/components/CritiqueBrainzLinks.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DBDefs from '../static/scripts/common/DBDefs';
+import {l, ln} from '../static/scripts/common/i18n';
+
+const seeReviewsHref = (releaseGroup) => (
+  DBDefs.CRITIQUEBRAINZ_SERVER +
+  '/release-group/' +
+  releaseGroup.gid
+);
+
+const writeReviewLink = (releaseGroup) => (
+  DBDefs.CRITIQUEBRAINZ_SERVER +
+  '/review/write?release_group=' +
+  releaseGroup.gid
+);
+
+type Props = {|
+  +releaseGroup: ReleaseGroupT,
+|};
+
+const CritiqueBrainzLinks = ({releaseGroup}: Props) => {
+  const reviewCount = releaseGroup.review_count;
+
+  if (reviewCount === 0) {
```

Comment:
The way this is written looks like we want it to be displayed always, but with the "write a review" link if no reviews exist. Yet on the release sidebar, we only show it if releaseGroup.review_count is true. Which seemingly means it does not get shown unless there are already some reviews, and this section is apparently pointless. We should probably either remove this or always display it (I'd go for always display it).

This is already the case, so it's not introducing a regression or anything, but we might as well change it while we're at it?